### PR TITLE
Add LUKSO configurations

### PR DESCRIPTION
**Pull request by andocodes**
Add LUKSO chain specifics to settings
Build docker image from LUKSO google artifactory cloud.
Add CI/CD stuffs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -311,19 +311,19 @@ make build_docker
 Run the following command to enter the interactive CLI:
 
 ```sh
-docker run -it --rm -v $(pwd)/validator_keys:/app/validator_keys ethereum/staking-deposit-cli
+docker run -it --rm -v $(pwd)/validator_keys:/app/validator_keys europe-docker.pkg.dev/lks-lz-artifacts/docker-deposit-cli
```
Comment (frozeman) [inline README.md:314]: Why hosting this on docker pkg? and where is it hosted at ethereum? directly in the repo?

File: staking_deposit/cli/generate_keys.py [hunk 2]
```diff
@@ -121,10 +127,10 @@ def generate_keys_arguments_decorator(function: Callable[..., Any]) -> Callable[
 @click.pass_context
 def generate_keys(ctx: click.Context, validator_start_index: int,
                   num_validators: int, folder: str, chain: str, keystore_password: str,
-                  eth1_withdrawal_address: HexAddress, **kwargs: Any) -> None:
+                  amount: int, eth1_withdrawal_address: HexAddress, **kwargs: Any) -> None:
     mnemonic = ctx.obj['mnemonic']
     mnemonic_password = ctx.obj['mnemonic_password']
-    amounts = [MAX_DEPOSIT_AMOUNT] * num_validators
+    amounts = amount * num_validators
```
Comment (frozeman) [inline staking_deposit/cli/generate_keys.py:133]: I guess this max amount we can hardcode, once decided

File: staking_deposit/settings.py [hunk 3]
```diff
@@ -27,7 +29,10 @@ class BaseChainSetting(NamedTuple):
 KilnSetting = BaseChainSetting(NETWORK_NAME=KILN, GENESIS_FORK_VERSION=bytes.fromhex('70000069'))
 # Sepolia setting
 SepoliaSetting = BaseChainSetting(NETWORK_NAME=SEPOLIA, GENESIS_FORK_VERSION=bytes.fromhex('90000069'))
-
+# LUKSO L16 testnet setting
+LUKSOL16Setting = BaseChainSetting(NETWORK_NAME=L16_TESTNET, GENESIS_FORK_VERSION=bytes.fromhex('60000069'))
+# LUKSO mainnet setting - change when mainnet parameters are set and launched - for now reverts to L16
+LUKSOSetting = BaseChainSetting(NETWORK_NAME=LUKSO, GENESIS_FORK_VERSION=bytes.fromhex('60000069'))
```
Comment (frozeman) [inline staking_deposit/settings.py:35]: TODO: change to `0x38380001 `

File: README.md [hunk 4]
```diff
@@ -311,19 +311,19 @@ make build_docker
...
 ```sh
-docker run -it --rm -v $(pwd)/validator_keys:/app/validator_keys ethereum/staking-deposit-cli new-mnemonic --num_validators=<NUM_VALIDATORS> --mnemonic_language=english --folder=<YOUR_FOLDER_PATH>
+docker run -it --rm -v $(pwd)/validator_keys:/app/validator_keys europe-docker.pkg.dev/lks-lz-artifacts/docker-deposit-cli new-mnemonic --num_validators=<NUM_VALIDATORS> --mnemonic_language=english --folder=<YOUR_FOLDER_PATH>
 ```
 
 Example for 1 validator on the [Prater testnet](https://prater.launchpad.ethereum.org/) using english:
 
 ```sh
-docker run -it --rm -v $(pwd)/validator_keys:/app/validator_keys ethereum/staking-deposit-cli new-mnemonic --num_validators=1 --mnemonic_language=english --chain=prater
+docker run -it --rm -v $(pwd)/validator_keys:/app/validator_keys europe-docker.pkg.dev/lks-lz-artifacts/docker-deposit-cli new-mnemonic --num_validators=1 --mnemonic_language=english --chain=prater
```
Comment (blazejkrzak) [inline README.md:326]: you forgot to change the `--chain` flag value in the readme

File: staking_deposit/settings.py (hunk 3 above)
Comment (blazejkrzak) [reply staking_deposit/settings.py:35]: yeah, lets settle this asap

**Pull request merged**